src/travel-details-screens/TripDetailsScreenComponent.tsx

```diff
@@ -186,7 +186,7 @@ function useGetTicketInfoFromTrip(tripPattern: TripPattern) {
   if (!(enable_ticketing && !someLegsAreNotSingleTicket)) return;
 
   const tripStartWithBuffer = addMinutes(
-    parseISO(tripPattern.expectedStartTime),
+    parseISO(nonFootLegs[0]?.aimedStartTime),
```

Comment:
The `parseISO` expect a string as input, but the result we are sending might be nil / undefined, we do not know what it responds when that happens.